editor/translations/editor/ar.po

```diff
@@ -16314,9 +16314,6 @@ msgstr "تعبير ثابت متوقع للوسيطة %d لاستدعاء الد
 msgid "Expected a boolean expression."
 msgstr "من المتوقع تعبير منطقي."
 
-msgid "Cases must be defined before default case."
-msgstr "يجب تحديد الحالات قبل الحالة الافتراضية."
-
```

Comment:
It's correct that those translations aren't needed anymore, but we don't modify translations directly in this repo, as those are just copies of what's being translated on Weblate: https://hosted.weblate.org/projects/godot-engine/godot/

They will be removed automatically next time I sync the translations, so for this commit you can undo the .po changes to reduce the diff.